src/gf_ring.h

```diff
@@ -349,12 +349,15 @@ void RingModN<T>::mul_coef_to_buf(T a, T* src, T* dest, size_t len)
         dest[i] = T((coef * src[i]) % this->_card);
     }
 }
+
```

Comment:
Maybe we should group all the SIMD specialization declaration in order to have a single ifdef block in this file. That would be more readable/easier to maintain.